### PR TITLE
Return JSON for accountabilities and domains on the Role model

### PR DESCRIPTION
**Pull request by redmerloen**
In the Role serializer return the accountabilities and domains fields as a JSON

### CODE REVIEW

**Comment by ghost** [comment]
* Please remove the TODO comments from the Model
* Please add docstring to the RoleSerializer (if we add documentation to everything we touch we'll get to 100%  documentation eventually ;) )
* json.loads() takes a string, but does it also do any validation? Cause I can still see this causing trouble for stings with a ' in them. And does it make the most sense to do the validation here? 
* The SerializerMethodField is read-only so doesn't actually change anything?

File: backend/flindt/role/serializers.py [hunk 1]
```diff
@@ -25,6 +40,18 @@ class Meta:
 
 class RoleSerializer(serializers.ModelSerializer):
```
Comment (noesnaterse) [inline backend/flindt/role/serializers.py:59]: No docstring?

File: backend/flindt/role/serializers.py [hunk 2]
```diff
@@ -25,6 +40,18 @@ class Meta:
...
     parent = ParentRoleSerializer()
+    accountabilities = serializers.SerializerMethodField()
+    domains = serializers.SerializerMethodField()
+
+    def get_accountabilities(self, obj):
+        if obj.accountabilities:
```
Comment (noesnaterse) [inline backend/flindt/role/serializers.py]: Docstring.

File: backend/flindt/role/serializers.py [hunk 3]
```diff
@@ -25,6 +40,18 @@ class Meta:
...
+            return json.loads(obj.accountabilities)
+        return obj.accountabilities
+
+    def get_domains(self, obj):
```
Comment (noesnaterse) [inline backend/flindt/role/serializers.py]: And you guessed it...

**Review by noesnaterse — DISMISSED** [review]
Please add the docstrings.

**Comment by redmerloen** [comment]
@RosienVT:
The accountabilities are already stored as string version that is output from json.dumps(); This is done on this line: https://github.com/wearespindle/flindt/blob/develop/backend/flindt/integrations/importer.py#L339

for example this now returns a correct array of accountabilities:
```
id: 7,
name: "Facilitator",
purpose: "Circle governance and operational practices aligned with the Constitution.",
accountabilities: [
"Facilitating the Circle’s constitutionally-required meetings",
"Auditing the meetings and records of Sub-Circles as needed, and declaring a Process Breakdown upon discovering a pattern of behavior that conflicts with the rules of the Constitution"
],
```
this is the current develop:
which returns the accountabilities as string with an array.
```
id: 7,
name: "Facilitator",
purpose: "Circle governance and operational practices aligned with the Constitution.",
accountabilities: "["Facilitating the Circle\u2019s constitutionally-required meetings", "Auditing the meetings and records of Sub-Circles as needed, and declaring a Process Breakdown upon discovering a pattern of behavior that conflicts with the rules of the Constitution"]",
domains: "",
```

File: backend/flindt/role/serializers.py [hunk 4]
```diff
@@ -11,18 +11,36 @@ class ParentRoleSerializer(serializers.ModelSerializer):
...
     """
+    # Override the default accountabilities serializer with SerializerMethodField to override the getter.
     accountabilities = serializers.SerializerMethodField()
+    # Override the default domains serializer with SerializerMethodField to override the getter.
     domains = serializers.SerializerMethodField()
 
     def get_accountabilities(self, obj):
-        if obj.accountabilities:
+        """
+        Override the getter for domains to try and convert the domains to a
```
Comment (ghost) [inline backend/flindt/role/serializers.py]: domains = accountabilities? (even mierenneuken ;) )

**Pull request merged**